dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -96,6 +97,9 @@ message TopicEventRequest {
   // The matching path from TopicSubscription/routes (if specified) for this event.
   // This value is used by OnTopicEvent to "switch" inside the handler.
   string path = 9;
+
+  // The map of additional custom properties to be sent to the app. These are considered to be cloud event extensions.
+  google.protobuf.Struct extensions = 10;
```

Comment:
We have long discussion about this, since you don't want to limit the value to a single scalar type, then google.protobuf.Struct is Ok